app/models/github.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
```

Comment:
lets get rid of all f this

Reply:
oops. Completely forgot to delete this - that's what I get for coding over lunch